db/migrate/20211108134601_further_core_migrations.rb

```diff
@@ -0,0 +1,58 @@
+class FurtherCoreMigrations < ActiveRecord::Migration[6.1]
+  def up
+    change_table :case_logs, bulk: true do |t|
+      t.remove :condition_effects_prefer_not_to_say
+      t.remove :reason_for_leaving_last_settled_home
+      t.column :reason, :integer
+      t.remove :property_reference
+      t.column :propcode, :string
+      t.remove :property_major_repairs
+      t.column :majorrepairs, :integer
+      t.remove :property_location
+      t.column :la, :string
+      t.remove :previous_la
+      t.column :prevloc, :string
+      t.remove :housing_benefit
+      t.column :hb, :integer
+      t.remove :outstanding_rent_or_charges
+      t.column :hbrentshortfall, :integer
+      t.remove :outstanding_amount
+      t.column :tshortfall, :integer
+      t.column :postcode, :string
+      t.column :postcod2, :string
+      t.column :ppostc1, :string
+      t.column :ppostc2, :string
+      t.remove :property_relet
+      t.column :property_relet, :integer
+    end
+  end
+
+  def down
```

Comment:
This isn't currently reversible, I think because of the order columns are being added and removed in

```
$ rake db:rollback STEP=1
== 20211108134601 FurtherCoreMigrations: reverting ============================
-- change_table(:case_logs, {:bulk=>true})
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

PG::DuplicateColumn: ERROR:  column "condition_effects_prefer_not_to_say" of relation "case_logs" already exists
/home/dan/Documents/mhclg-data-collection-beta/db/migrate/20211108134601_further_core_migrations.rb:31:in `down'

Caused by:
ActiveRecord::StatementInvalid: PG::DuplicateColumn: ERROR:  column "condition_effects_prefer_not_to_say" of relation "case_logs" already exists
/home/dan/Documents/mhclg-data-collection-beta/db/migrate/20211108134601_further_core_migrations.rb:31:in `down'

Caused by:
PG::DuplicateColumn: ERROR:  column "condition_effects_prefer_not_to_say" of relation "case_logs" already exists
/home/dan/Documents/mhclg-data-collection-beta/db/migrate/20211108134601_further_core_migrations.rb:31:in `down'
Tasks: TOP => db:rollback

```

Reply:
fixed